cli/azd/cmd/container.go

```diff
@@ -544,4 +544,5 @@ func registerCommonDependencies(container *ioc.NestedContainer) {
 
 	registerAction[*provisionAction](container, "azd-provision-action")
 	registerAction[*downAction](container, "azd-down-action")
+	registerAction[*configShowAction](container, "azd-config-show-action")
```

Comment:
Do we need this here? Typically, we only needed explicitly register commands when we were using aliases with the same action.

Reply:
calling `azd config list` fails if this is missing.
That's because `azd config list` uses an action which requires the `azd-config-show-action` to be created